xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/NotificationsDisplayerUIX.xml

```diff
@@ -679,15 +679,20 @@
   &lt;li&gt;
     #set ($discard = $xwiki.jsx.use('XWiki.Notifications.Code.NotificationsDisplayerUIX'))
     #set ($discard = $xwiki.ssx.use('XWiki.Notifications.Code.NotificationsDisplayerUIX'))
-    &lt;div class="notifications-header clearfix"&gt;
-      &lt;div class="col-xs-6"&gt;
-        &lt;p&gt;&lt;strong&gt;$escapetool.xml($services.localization.render('notifications.menu.header'))&lt;/strong&gt;&lt;/p&gt;
-      &lt;/div&gt;
-      &lt;div class="col-xs-6 text-right"&gt;
-        &lt;p&gt;&lt;a href="$xwiki.getURL($xcontext.userReference, 'view', 'category=notifications')" class="notifications-settings"&gt;$services.icon.renderHTML('cog')&amp;nbsp;$escapetool.xml($services.localization.render('notifications.menu.header.settings'))&lt;/a&gt;&lt;/p&gt;
-      &lt;/div&gt;
-      &lt;div class="notifications-header-uix col-xs-12"&gt;&lt;/div&gt;
-    &lt;/div&gt;
+    &lt;div class="notifications-header clearfix">&#xd;
+    &lt;div class="col-xs-4">&#xd;
+    &lt;p>&lt;strong>$escapetool.xml($services.localization.render('notifications.menu.header'))&lt;/strong>&lt;/p>&#xd;
+    &lt;/div>&#xd;
+    &lt;div class="col-xs-8 text-right">&#xd;
+    &lt;p>&#xd;
+    &lt;a target="_blank" href="$xwiki.getURL('XWiki.Notifications.Code.NotificationRSSService', 'get', 'outputSyntax=plain')">&#xd;
+    $services.icon.renderHTML("rss")&amp;nbsp;$escapetool.xml($services.localization.render('notifications.rss.feedLink'))&lt;/a>&#xd;
```

Comment:
I prefer `'rss'` over `"rss"` because double quotes are interpreted (if there is a `$something` in it, it's gonna be displayed) meanwhile simple quotes are not.